examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -528,10 +528,11 @@ class SubscribeEvent : public ReportCommand
         ReportCommand("subscribe-event", credsIssuerConfig),
         mClusterIds(1, clusterId), mEventIds(1, eventId)
     {
-        AddArgument("attr-name", eventName);
-        AddArgument("min-interval", 0, UINT16_MAX, &mMinInterval);
-        AddArgument("max-interval", 0, UINT16_MAX, &mMaxInterval);
-        AddArgument("keepSubscriptions", 0, 1, &mKeepSubscriptions);
+        AddArgument("attr-name", eventName, 0, "Event name.");
+        AddArgument("min-interval", 0, UINT16_MAX, &mMinInterval, 0, "The requested minimum interval boundary floor in seconds.");
```

Comment:
Not sure what "floor" means in this case. Isn't "minimum" sufficient to describe the purpose?